demonstrator/ogc-api-processes/main.js

```diff
@@ -0,0 +1,201 @@
+const oapiProcessesUrl = "../processes/";
+let processId = '';
```

Comment:
is initialiasation with empty string needed?
if yes a short explanation would be great

```suggestion
let processId;
```

Reply:
its not needed, but i personally do like to initiliaze it as empty string so that potential string concatenations like e.g. `processId += ' finished'` does not fail which it would when the variable is not initialized.